### PR TITLE
Rachana_CoadingExam_31Aug_Done_UsingCollection

### PR DESCRIPTION
**Pull request by RachanaGJun20**
image

### CODE REVIEW

File: src/rachanaGJun20/CoadingExam/ATMPinValidate.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+		}
+		return false;
+	}
+
+	// check for digit
+	boolean isDigit(String pinnumber) {
+		try {
+			Integer.parseInt(pinnumber);
+
+		} catch (NumberFormatException e) {
```
Comment (KrishnaKTechnocredits) [inline src/rachanaGJun20/CoadingExam/ATMPinValidate.java:62]: I was expecting this. good job

File: src/rachanaGJun20/CoadingExam/ATMPinValidateWithoutCollection.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+				}
+			} else {
+				System.out.println("Number is not of valid length " + pinnumber);
+			}
+		}
+	}
+
+	private boolean isRepeatedDigit(String pinnumber) {
+		char[] charray = pinnumber.toCharArray();
+		for (int index = 0; index < charray.length; index++) {
```
Comment (KrishnaKTechnocredits) [inline src/rachanaGJun20/CoadingExam/ATMPinValidateWithoutCollection.java:30]: 2 loops are not required..
first number should be verified with exactly next number.. and so on.. dont you think so ...

**Review by KrishnaKTechnocredits — APPROVED** [review]
Code merge !!

File: src/rachanaGJun20/CoadingExam/ATMPinValidate.java (hunk 1 above)
Comment (RachanaGJun20) [reply src/rachanaGJun20/CoadingExam/ATMPinValidate.java:62]: Thank you :)

File: src/rachanaGJun20/CoadingExam/ATMPinValidateWithoutCollection.java (hunk 2 above)
Comment (RachanaGJun20) [reply src/rachanaGJun20/CoadingExam/ATMPinValidateWithoutCollection.java:30]: Yes , Got your point......  Will change it.
Thank you.

**Pull request merged**